packages/cozy-konnector-libs/libs/linkBankOperations.js

```diff
@@ -87,9 +131,10 @@ class Linker {
       const res = result[bill._id] = { bill:bill }
 
       const linkBillToDebitOperation = () => {
-        return findDebitOperation(this.cozyClient, bill, options)
+        return findDebitOperation(this.cozyClient, bill, options, allOperations)
           .then(operation => {
             if (operation) {
+              console.log('found debit operation', operation)
```

Comment:
Is it a temporary `console.log` here ? If not, maybe use `log` ?

Reply:
thx